Client/Frontend/Browser/TabGroupData.swift

```diff
@@ -0,0 +1,72 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/
+
+import Foundation
+import MozillaAppServices
+
+public enum TabGroupTimerState: String, Codable {
+    case navSearchLoaded
+    case tabNavigatedToDifferentUrl
+    case tabSwitched
+    case tabSelected
+    case newTab
+    case openInNewTab
+    case openURLOnly
+    case none
+}
+
+public class TabGroupData: NSObject, NSCoding {
+    var tabAssociatedSearchTerm: String = ""
+    var tabAssociatedSearchUrl: String = ""
+    var tabAssociatedNextUrl: String = ""
+    var tabHistoryCurrentState = ""
+    // Is not used currently can be removed? Name is confusing with enum
+    var tabGroupTimerState = ""
```

Comment:
So could we remove it? Would be nice either to remove now or create a tech task for us to not forget this

Reply:
I talk with @nbhasin2 about this one, it was created with a purpose but it's not currently used so we could remove it and add it back if is needed in the future